### ISSUE TITLE
"How 🤗 Transformers solve tasks" chapter's ViT explaination is jumbled up in steps

### ISSUE THREAD

**Issue by BhavyaShah1234** [huggingface/course#933]
`
The main change ViT introduced was in how images are fed to a Transformer:

1. An image is split into square non-overlapping patches, each of which gets turned into a vector or patch embedding. The patch embeddings are generated from a convolutional 2D layer which creates the proper input dimensions (which for a base Transformer is 768 values for each patch embedding). If you had a 224x224 pixel image, you could split it into 196 16x16 image patches. Just like how text is tokenized into words, an image is “tokenized” into a sequence of patches.

2. A learnable embedding - a special [CLS] token - is added to the beginning of the patch embeddings just like BERT. The final hidden state of the [CLS] token is used as the input to the attached classification head; other outputs are ignored. This token helps the model learn how to encode a representation of the image.

3. The last thing to add to the patch and learnable embeddings are the position embeddings because the model doesn’t know how the image patches are ordered. The position embeddings are also learnable and have the same size as the patch embeddings. Finally, all of the embeddings are passed to the Transformer encoder.

4. The output, specifically only the output with the [CLS] token, is passed to a multilayer perceptron head (MLP). ViT’s pretraining objective is simply classification. Like other classification heads, the MLP head converts the output into logits over the class labels and calculates the cross-entropy loss to find the most likely class.
`

The above explaination of Vision Transformer is jumbled up in steps. Please fix that as soon as possible as it causes some confusion in understanding for beginners

Link: [https://huggingface.co/learn/llm-course/chapter1/5?fw=pt#image-classification](https://huggingface.co/learn/llm-course/chapter1/5?fw=pt#image-classification)
